### ISSUE TITLE
Thor/Delta avionics from FASA lacks EC

### ISSUE THREAD

**Issue by vallin91** [KSP-RO/RealismOverhaul#1967]
Since I installed, what was yesterday, the latest RO master I've noticed that the Thor/Delta avionics from FASA pack is completely lacking electric charge.

I don't know if this is intentional, but I thought I should bring it up.

![ksp_x64_2018-12-30_04-53-49](https://user-images.githubusercontent.com/46173191/50544222-0a55df80-0bef-11e9-9d8f-4230c16fcf5d.png)
![ksp_x64_2018-12-30_04-54-09](https://user-images.githubusercontent.com/46173191/50544224-0e81fd00-0bef-11e9-94b6-bdbee049c7a3.png)

**Comment by Bornholio** [comment]
probably should be a FASA issue unless RO additionally change it in a separate way.

**Comment by Bornholio** [comment]
line 992 
```
	RESOURCE[ElectricCharge]
	{
		name = ElectricCharge // Again using values from 1m early ring
		amount = 108 // 6 minutes
		maxAmount = 108
	}
```

**Comment by Bornholio** [comment]
changing RESOURCE[ElectricCharge] to RESOURCE
https://cdn.discordapp.com/attachments/331813459417235457/530200202380836866/unknown.png
https://cdn.discordapp.com/attachments/331813459417235457/530200502470836262/unknown.png

**Comment by vallin91** [comment]
This worked, thanks for the help.

**Comment by Bornholio** [comment]
closed by #1970
